src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -326,17 +304,12 @@ protected override void Dispose(bool disposing)
                 {
                     base.Dispose(disposing);
 
-                    if (_start != null)
+                    if (disposing)
                     {
-                        _accessor.SafeMemoryMappedViewHandle.ReleasePointer();
-                        _start = null;
+                        _handle.Dispose();
```

Comment:
:arrow_right: `_handle` is a managed object. Interactions with managed objects during `Dispose` are always¹ conditional on `disposing` being true.

¹ There are limited exceptions, but none apply to anything related to this code so there's nothing to get confused about.